packages/ng/form-field/form-field.component.html

```diff
@@ -49,7 +49,7 @@
 			<span class="formLabel-tag tag">{{ tag() }}</span>
 		}
 		@if (counter() > 0) {
-			<span class="formLabel-counter" [class.pr-u-textError]="contentLength > counter()" id="{{ id() }}-counter" aria-live="polite">
+			<span class="formLabel-counter" [class.pr-u-colorTextCritical]="contentLength > counter()" id="{{ id() }}-counter" aria-live="polite">
```

Comment:
Can we ensure backwards compatibility with the current naming if we change the classes in the component's template?